editor.planx.uk/src/@planx/components/FileUploadAndLabel/Public.tsx

```diff
@@ -130,19 +130,26 @@ function Component(props: Props) {
 
   return (
     <Card
-      handleSubmit={handleSubmit}
-      isValid={slots.every((slot) => slot.url && slot.status === "success")}
+      handleSubmit={props.hideDropZone ? props.handleSubmit : handleSubmit}
```

Comment:
This reads a little funny (but is perfectly correct!)

This is because `handleSubmit()` is not a very accurate / descriptive name - maybe we need something more meaningful like `validateAndSubmit()`. We could also destructure `props.handleSubmit` to a more meaningful name (maybe overkill?).

`handleSubmit={props.hideDropZone ? continueWithoutUploads : validateUploadsAndSubmit}`

Reply:
Totally agree this is awkward - renaming the custom `handleSubmit` we've defined in the component to `validateAndSubmit()` makes most sense to me :+1: 

For the sake of global search-ability, I think it's actually ideal _not_ to destructure/rename props.handleSubmit to something else!

Reply:
Yep - I agree here, this works!